curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-asynchronous-programming-by-building-an-fcc-forum-leaderboard/645f7879ebbdb201892e55e1.md

```diff
@@ -0,0 +1,353 @@
+---
+id: 645f7879ebbdb201892e55e1
+title: Step 71
+challengeType: 0
+dashedName: step-71
+---
+
+# --description--
+
+In this step, you will construct the `avatarUrl` by checking if the `avatar` string starts with `/user_avatar/`. If it does, you will concatenate it with the `avatarUrl`. If it doesn't, you will simply use the `avatar` as it is. This will ensure the avatar URL is correctly formed whether it's a relative or absolute URL. You can use a ternary operator to accomplish this.
+
+# --hints--
+
+You should create a constant named `avatarUrl`. Use the `startsWith` method on the `avatar` string to check if it begins with `/user_avatar/`. If this condition is true, concatenate `avatarUrl` with `avatar`. Otherwise, simply use `avatar`.
```

Comment:
```suggestion
You should create a constant named `userAvatarURL `. Use the `startsWith` method on the `avatar` string to check if it begins with `/user_avatar/`. If this condition is true, concatenate `avatarUrl` with `avatar`. Otherwise, simply use `avatar`.
```